testslide/runner.py

```diff
@@ -279,6 +279,122 @@ def finish(self, not_executed_examples):
             self.print_cyan("  Not executed: ", len(not_executed_examples))
 
 
+class LongFormatter(Formatter):
+    def new_example(self, example):
+        self.print_white(
+            "{}{}: ".format(
+                "*" if example.context.focus else "", example.context.full_name
+            ),
+            end="",
+        )
+
+    def _color_output(self):
+        return sys.stdout.isatty() or self.force_color
+
+    def success(self, example):
+        Formatter.success(self, example)
+        self.print_green(
+            "{focus}{example}{pass_text}".format(
+                focus="*" if example.focus else "",
+                example=example,
+                pass_text="" if self._color_output() else ": PASS",
+            )
+        )
+
+    def fail(self, example, exception):
+        if isinstance(exception, AggregatedExceptions) and 1 == len(
+            exception.exceptions
+        ):
```

Comment:
Ignorance of the codebase (my bad): is there a chance that `... and len(exception.exceptions) > 1` ?